_inc/client/at-a-glance/monitor.jsx

```diff
@@ -18,29 +19,47 @@ import { getLastDownTime as _getLastDownTime } from 'state/at-a-glance';
 
 const DashMonitor = React.createClass( {
 	getContent: function() {
+		const labelName = __( 'Downtime Monitoring' );
+
 		if ( this.props.isModuleActivated( 'monitor' ) ) {
 			const lastDowntime = this.props.getLastDownTime();
 
 			if ( lastDowntime === 'N/A' ) {
 				return(
-					<DashItem label="Downtime Monitoring" status="is-working">
+					<DashItem label={ labelName } status="is-working">
 						<QueryLastDownTime />
-						<p className="jp-dash-item__description">Loading&#8230;</p>
+						<p className="jp-dash-item__description">{ __( 'Loading...' ) }</p>
 					</DashItem>
 				);
 			}
 
 			return(
-				<DashItem label="Downtime Monitoring" status="is-working">
-					<p className="jp-dash-item__description">Monitor is on and is watching your site.</p>
-					<p className="jp-dash-item__description">Last downtime was { lastDowntime.date } ago.</p>
+				<DashItem label={ labelName } status="is-working">
+					<p className="jp-dash-item__description">{ __( 'Monitor is on and is watching your site.' ) }</p>
+					<p className="jp-dash-item__description">
+						{
+							__( 'Last downtime was %(time)s ago.', {
+								args: {
+									time: lastDowntime.date
```

Comment:
we're using core's `human_time_diff()` [#](https://codex.wordpress.org/Function_Reference/human_time_diff) to get this PHP side... Wondering how translations work with that method.